pkg/driver/camera/camera_linux.go

```diff
@@ -83,6 +83,8 @@ func (c *camera) Open() error {
 		return err
 	}
 
+	// Late frames should be discarded. Buffering should be handled in higher level.
+	cam.SetBufferCount(1)
```

Comment:
I guess dual buffer is required to load next frame during using previous frame. (not very sure it's required)

Reply:
From my test, this seems to be fine. I also tried with double buffer, but it has higher latency.